sound/soc/sof/intel/hda-dsp.c

```diff
@@ -23,7 +23,7 @@
 #include "hda.h"
 #include "hda-ipc.h"
 
-static bool hda_enable_trace_D0I3_S0;
+static bool hda_enable_trace_D0I3_S0 = true;
```

Comment:
should be default false to match the code, this needs to be set as true for debug cases where you want to enable the trace in S0/D0i3.

Reply:
@plbossart @keyonjie @ranj063  Hmm, I'm also not really clear whether we really have a need for this rich interface to control via kernel params. I'd imagine for most case, simple check of:
```
	if (!sdev->dtrace_is_supported ||
	    sdev->system_suspend_target != SOF_SUSPEND_NONE)
			flags = HDA_PM_NO_DMA_TRACE;
```

... would cover most needs without any extra kernel params.

But alas, my mileage debugging d0ix issues pales in comparison to you folks, so I'll take your word on this. I do think the default should be "false".  D0ix with DMA active for traces is a special state, so should not be the default.